templates/compositions/ec-component-navigation-list/navigation-list.html.twig

```diff
@@ -37,5 +38,16 @@
       {% endfor %}
     </ul>
   {% endif %}
+  {% if secondary_links is not empty and secondary_links is iterable %}
+    <div class="ecl-u-mt-m ecl-u-border-top ecl-u-border-color-grey-50"">
+      <ul class="ecl-u-mt-m ecl-unordered-list ecl-unordered-list--no-bullet">
+        {% for link in secondary_links %}
+          <li class="ecl-unordered-list__item">
+            {% include '@ecl-twig/link' with { link: link.link|merge({type: 'standalone'}) } only %}
```

Comment:
We already do setting of 'type' here: https://github.com/openeuropa/oe_theme/pull/490/files#diff-8ac0e3155ae22e8e48044f5588bbffaeR26